tests/com/goide/editor/GoHighlightingAnnotatorTest.java

```diff
@@ -62,6 +62,14 @@ public void testBuiltinFunctions() {
     doTest();
   }
 
+  public void testStringSliceWithThirdIndex() {
```

Comment:
wrong place. as you can see by testDirectory all these tests are about color highlighting, not errors highlighting